tests/datadriven/test_references.py

```diff
@@ -83,12 +84,13 @@ def testValidateObjects(self):
         # test that validation works on reference sets and references
         referenceSet = self._gaObject
         referenceSetPe = referenceSet.toProtocolElement()
-        self.assertValid(
-            protocol.ReferenceSet, referenceSetPe.toJsonDict())
-        self.assertGreater(len(referenceSetPe.referenceIds), 0)
+        self.assertIsInstance(
+            referenceSetPe, protocol.ReferenceSet)
+        self.assertGreater(len(referenceSetPe.reference_ids), 0)
         for gaReference in referenceSet.getReferences():
-            reference = gaReference.toProtocolElement().toJsonDict()
-            self.assertValid(protocol.Reference, reference)
```

Comment:
We're making these tests weaker here. Why not leave them as assertValid? We can convert them to JSON, and convert them back as before. Won't proto throw an error if this fails?